packages/fluentui/react-northstar/src/themes/teams/components/Popup/popupContentVariables.ts

```diff
@@ -21,15 +23,17 @@ export interface PopupContentVariables {
 
 export const popupContentVariables = (siteVars: any): PopupContentVariables => {
   return {
-    borderColor: siteVars.colorScheme.default.border2,
+    autoSizeBorderColor: siteVars.colorScheme.default.border2,
```

Comment:
I don't see any usage of this variable. Cruft?

Reply:
Maybe a bad merge. Thanks.